packages/helper-grammar-regex-collection/index.js

```diff
@@ -73,6 +73,10 @@ export const DOCKER_FROM = regex`
   FROM \s (?<$1>[^\n]*)
 `;
 
+export const DOCKER_ENTRYPOINT = regex`
+  ENTRYPOINT \s \[${captureQuotedWord}\]
+`;
+
```

Comment:
Could you add some tests cases to verify that this RegExp is working properly, please. 

https://github.com/OctoLinker/OctoLinker/blob/master/packages/helper-grammar-regex-collection/test.js

Reply:
Definitely, I just added them. Let me know if there are any cases I might be missing here.